sh/tests/division_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+challenge() {
+    submitted=$(bash "$script_dirS"/student/division.sh $1 $2)
+    expected=$(bash "$script_dirS"/solutions/division.sh $1 $2)
+
+    diff <(echo "$submitted") <(echo "$expected")
+}
+
+challenge "10" "2"
+challenge "4" "2"
+challenge "0.5" "0.5"
+challenge "5" "2"
+challenge "0.5"
+challenge "foo" "bar"
```

Comment:
this never fails the test as well even if the student don't have the correct error message
so i guess its not even testing

Reply:
Thanks, I realised that I was testing in a wrong way, because the solution was exiting with 1 it wasn't outputting the diff and it would always fail. I was testing it, but I suspect that because of the exit status 1 it was never getting to the point to display the diff. I'll modify the exercise to make it work with an exit 0